lesson-5/script.js

```diff
@@ -0,0 +1,104 @@
+// //Вивести в console числа, від 10 до 25 з кроком 0.5 (10 й 25 включно)
+for (i = 10; i <= 25; i += 0.5) {
+    console.log(i)
+}
+// Запитати та зберегти число введене із prompt.
+// Визначати чи це число просте
+const someNumberFromUser = +prompt("Enter some positive number here, please.")
+if (someNumberFromUser === 1) {
+    console.log(`${someNumberFromUser} is neither prime nor composite number.`);
+} else if (someNumberFromUser === 2) {
+    console.log(`${someNumberFromUser} is a prime number.`);
+} else if (someNumberFromUser > 2) {
+    for (let i = 2; i < someNumberFromUser; i++) {
+        if (someNumberFromUser % i == 0) {
+            console.log(`${someNumberFromUser} is not a prime number`);
+            break;
+        }
+        else {
+            console.log(`${someNumberFromUser} is a prime number`);
+            break;
+        }
+    }
+} else {
+    console.log(`${someNumberFromUser} is not a prime number.`);
+}
+/* Alternate decision is below
+const someNumberFromUser = +prompt("Enter some positive number here, please.");
+let isPrime = true;
+if (someNumberFromUser === 1) {
+    console.log("1 is neither prime nor composite number.");
+}
+else if (someNumberFromUser > 1) {
+    for (let i = 2; i < someNumberFromUser; i++) {
+        if (someNumberFromUser % i == 0) {
+            isPrime = false;
+            break;
+        }
+    }
+    if (isPrime) {
+        console.log(`${someNumberFromUser} is a prime number`);
+    } else {
+        console.log(`${someNumberFromUser} is not a prime number`);
+    }
+}
+else {
+    console.log("The number is not a prime number.");
+}
+*/
+
+// /*Запитати та зберегти суму покупки введене із prompt. Обрахувати знижку та повідомити користувачу значення знижки. 
+// Якщо сума покупки до 100 UAН - знижка 3%; якщо сума покупки до 200 UAH - знижка 5%; якщо сума покупки перевищує 200 UAH - знижка 7%*/
+const purchasePrice = +prompt("What is your last purchase price in UAH?");
+if (purchasePrice < 100) {
+    console.log(`Your discount is 3%`);
+} else if ( purchasePrice < 200) {
+    console.log(`Your discount is 5%`);
+} else {
+    console.log(`Your discount is 7%`);
+}
+// Вивести таблицю множення чисел від 2 до 9 у зрозумілому форматі
+for (i = 2; i < 10; i++) {
+    for (j = 2; j < 10; j++) {
```

Comment:
```suggestion
    for (j = 1; j <= 10; j++) {
```
 Multiplication table normally starts with 1 and ends with 10 multiplication

Reply:
it was a condition: "...від 2 до 9..."

Reply:
My bad, it should be 
`for (j = 1; j <= 9; j++) {`